rotkehlchen/tests/utils/dataimport.py

```diff
@@ -2514,3 +2514,272 @@ def assert_kucoin_import_results(rotki: Rotkehlchen):
+            timestamp=Timestamp(1673194920),
+            address=None,
+            transaction_id=None,
+            asset=A_BNB,
+            amount=AssetAmount(FVal('0.00915109')),
+            fee_asset=A_USD,
+            fee=Fee(ZERO),
+            link='',
+        ), AssetMovement(
+            location=Location.EXTERNAL,
+            category=AssetMovementCategory.WITHDRAWAL,
+            timestamp=Timestamp(1673204160),
+            address=None,
+            transaction_id=None,
+            asset=A_EUR,
+            amount=AssetAmount(FVal('150')),
+            fee_asset=A_USD,
+            fee=Fee(ZERO),
+            link='',
+        ), AssetMovement(
+            location=Location.BITPANDA,
+            category=AssetMovementCategory.DEPOSIT,
+            timestamp=Timestamp(1674759960),
+            address=None,
+            transaction_id=None,
+            asset=A_EUR,
+            amount=AssetAmount(FVal('1000')),
+            fee_asset=A_USD,
+            fee=Fee(ZERO),
+            link='',
+        ), AssetMovement(
+            location=Location.BITPANDA,
+            category=AssetMovementCategory.WITHDRAWAL,
+            timestamp=Timestamp(1674760200),
+            address=None,
+            transaction_id=None,
+            asset=A_EUR,
+            amount=AssetAmount(FVal('1000')),
+            fee_asset=A_USD,
+            fee=Fee(ZERO),
+            link='',
+        ),
+    ]
+
+    expected_history_events = [
+        HistoryEvent(
+            identifier=1,
+            event_identifier='',
+            sequence_index=0,
+            timestamp=TimestampMS(1673035680000),
+            location=Location.EXTERNAL,
+            asset=A_ETH_MATIC,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            balance=Balance(
+                amount=FVal('0.001537148341223424'),
+                usd_value=ZERO,
+            ),
+            notes='Fee of 0.001537148341223424 MATIC in external',
+        ), HistoryEvent(
+            identifier=2,
+            event_identifier='',
+            sequence_index=1,
+            timestamp=TimestampMS(1673035680000),
+            location=Location.EXTERNAL,
+            asset=Asset('META'),
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.NONE,
+            balance=Balance(
+                amount=FVal('10'),
+                usd_value=ZERO,
+            ),
+            notes='Spend 10 META in external',
+        ), HistoryEvent(
+            identifier=3,
+            event_identifier='',
+            sequence_index=1,
+            timestamp=TimestampMS(1673139600000),
+            location=Location.BINANCE,
+            asset=A_EUR,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.NONE,
+            balance=Balance(
+                amount=FVal('0.01061899'),
+                usd_value=ZERO,
+            ),
```

Comment:
no need since the defaults are ZERO. Same for other places in this file where this happens

```suggestion
            balance=Balance(amount=FVal('0.01061899')),
```